### ISSUE TITLE
Accessibility - include aria attributes where appropriate

### ISSUE THREAD

**Issue by kaylaondracek** [mlibrary/fishrappr#176]
## Description: 

include aria attributes where appropriate throughout site

## Acceptance Criteria:

- [x] The search form should have aria-live="polite"
- [x] Browse form
- [x] Graph

**Comment by gordonleacock** [comment]
I've added aria-live="polite" to the search form.

Changes driven by clicking the graph trigger a page reload. Tabbing through the graph should provide accessible data to compensate for not seeing the pop up count but this needs testing.

On the browse page I am adding aria-live="polite" to the browse-select-box div to alert screen reader user of the updating to the date_issued_dd_ti select box (for example, if you select the month of February, that select is updated to 28 or 29 days, depending on leap years).

Testing and accessibility review may show other areas that need ARIA.
